config/controller.yaml

```diff
@@ -28,6 +28,7 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "false"
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
```

Comment:
I think we mainly want this for our data-plane (activator) and control-plane elements that sit on the data-plane (autoscaler, and maybe webhook).  The rest should recover quickly if evicted.  WDYT?

Reply:
I guess for controllers it's less important.

Reply:
Done.